client/packages/requisitions/src/RequestRequisition/ListView/ProgramRequisitionOptions.tsx

```diff
@@ -29,6 +33,28 @@ type Common<T> = Pick<
   labelNoOptions?: string;
 };
 
+export const getOrderTypeRenderer =
+  (): AutocompleteOptionRenderer<OrderTypeRowFragment> => (props, item) => (
+    <DefaultAutocompleteItemOption {...props} key={item.id}>
+      <Box display="flex" flexDirection="row" gap={1} alignItems="center">
+        {item?.isEmergency && (
+          <Box display="flex" alignItems="center">
+            <EmergencyIcon />
```

Comment:
I wonder if we could have this icon in the selected field also.

Currently reverts back to just text after selection and feels less intuitive for me.

![Screenshot 2024-12-05 at 2 24 25 PM](https://github.com/user-attachments/assets/dec9a145-a817-4880-bf10-8cf5b4fa5481)

Reply:
I think fine to leave this for carry over. I don't wanna spend too much time on this right now. Having the emergency icon shown in the selection to differentiation is probably more useful than seeing the icon after it's being selected. Since you already know that you have selected the emergency order...